docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -52,10 +52,54 @@ and :ref:`our facility choice module documentation <2024_vivarium_mncnh_portfoli
 2.0 Module Input and Output Data
 ++++++++++++++++++++++++++++++++
 
-2.1 Module Description 
+2.1 Module Inputs
+-----------------
+
+.. list-table:: Module required inputs
+  :header-rows: 1
+
+  * - Input
+    - Source module
+    - Application
+    - Note
+  * - Pregnancy term length
+    - :ref:`Pregnancy I module <2024_vivarium_mncnh_portfolio_pregnancy_module>`
+    - Determines which set of probabilities to use for the ANC exposure categories. 
+    - Full term pregnancies can be assigned any of the values A, B, C, or D, 
+      whereas we assume partial term pregnancies can only attend ANC in the first trimester, so only categories B and D have nonzero probability. 
+      See exposure probability tables below.
+  * - ANC propensity
+    - :ref:`Initial attributes module <2024_vivarium_mncnh_portfolio_initial_attributes_module>`
+    - A random value uniformly distributed in [0, 1], interpreted as the simulant's intrinsic propensity for attending ANC visits, with higher 
+      propensity values corresponding to seeking more ANC care. Use the ANC propensity together with the ANC attendance exposure probabilities below 
+      to select one of the ANC exposure categories A, B, C, or D, as described in the :ref:`special ordering of the categories <facility_choice_special_ordering_of_categories_section>` 
+      section of the facility choice model document. When sampling, the categories should be ordered D < C < B < A (highest risk to lowest risk in terms of ultrasound timing).
```

Comment:
```suggestion
    - Use the ANC propensity together with the ANC attendance exposure probabilities below 
      to select one of the ANC exposure categories A, B, C, or D, as described in the :ref:`special ordering of the categories <facility_choice_special_ordering_of_categories_section>` 
      section of the facility choice model document. When sampling, the categories should be ordered D < C < B < A (highest risk to lowest risk in terms of ultrasound timing), so that higher 
      propensity values correspond to seeking more ANC care.
```
I think we can remove the definition of the propensity since it applies generally to any of our propensities (though I don't think we've actually documented that anywhere...). Here's a proposed edit.